File: 01_VisualBasic/0０_写径/WindowsFormsApp1/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace WindowsFormsApp1
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            label1.Text = "入力された文字は"+textBox1.Text;
        }

        private void Form1_Load(object sender, EventArgs e)
        {
            listBox1.Items.Add("りんご");
            listBox1.Items.Add("バナナ");
            listBox1.Items.Add("いちご");
        }

        private void checkBox1_CheckedChanged(object sender, EventArgs e)
        {
            if (checkBox1.Checked){
                label1.Text = "チェックボックスがチェックされました";
                }
            else
            {
                label1.Text = "チェックボックスからチェックが外れました";
            }

        }
    }
}

File: windows/01_VisualBasic/単位変換アプリ/Form1.vb
Public Class Form1

    Dim Amount As Integer '変換前の長さ
    Dim AmountConverted As Integer　'返還後の長さ

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Amount = 0
        Amount = TextBox1.Text

        If ComboBox1.SelectedIndex = -1 Then '単位の入力がない場合のエラーメッセージ
            Label1.Text = "選択されていません"
        ElseIf ComboBox1.Text = "cm" Then 'cmをmに変換
            Amount = Amount / 100
        ElseIf ComboBox1.Text = "km" Then 'kmをmに変換
            Amount = Amount * 100
        End If

        Label1.Text = Amount

    End Sub

    Private Sub Form1_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        ComboBox1.Items.Add("cm")
        ComboBox1.Items.Add("m")
        ComboBox1.Items.Add("km")
    End Sub
End Class
